components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java

```diff
@@ -177,10 +177,23 @@ public Response loginAppNamePost(String appName
                     NewCookie accessTokenhttpOnlyCookie = AuthUtil
                             .cookieBuilder(SPConstants.WSO2_SP_TOKEN_2, accessTokenSecondHalf, appContext, true, true,
                                     -1);
-                    NewCookie logoutContextAccessToken = AuthUtil
-                            .cookieBuilder(AuthRESTAPIConstants.WSO2_SP_TOKEN, accessTokenSecondHalf,
-                                    AuthRESTAPIConstants.LOGOUT_CONTEXT + appContext, true, true,
-                                    -1);
+
+                    NewCookie logoutContextAccessToken;
+                    IdPClientConfiguration authConfigurations = DataHolder.getInstance()
+                            .getConfigProvider()
+                            .getConfigurationObject(IdPClientConfiguration.class);
+                    if (authConfigurations.isSsoEnabled()) {
+                        logoutContextAccessToken = AuthUtil
+                                .cookieBuilder(AuthRESTAPIConstants.WSO2_SP_TOKEN, accessTokenSecondHalf,
+                                        AuthRESTAPIConstants.LOGOUT_CONTEXT + AuthRESTAPIConstants.LOGOUT_SSO_CONTEXT +
+                                                appContext , true, true, -1);
+                    } else {
+                        logoutContextAccessToken = AuthUtil
+                                .cookieBuilder(AuthRESTAPIConstants.WSO2_SP_TOKEN, accessTokenSecondHalf,
+                                        AuthRESTAPIConstants.LOGOUT_CONTEXT + appContext, true, true,
+                                        -1);
```

Comment:
Shall we reformat this to the previous line?

Reply:
Fixed in 6ed65129793669c8b287520f081c75f00fd0e92f